### ISSUE TITLE
Can't compile using gcc 10.

### ISSUE THREAD

**Issue by cdn-lnk** [epics-modules/softGlueZynq#5]
Good day.

**[gcc 10 defaults to -fno-common](https://gcc.gnu.org/gcc-10/porting_to.html) which causes errors while compiling synApps due to "myISRData" being defined multiple times.**

**Steps to reproduce**
1. Download and build EPICS Base.
2. Download and build SynApps.
3. The error will occur naturally while building synApps.

**Expected behaivour**
No errors when compiling 🙂.

**Actual behaviour**
We got the follwing error:
```sh
make[4]: Entering directory '/opt/epics/synApps/support/softGlueZynq-R2-0-1/softGlueApp/src/O.linux-x86_64'
/usr/bin/g++ -o libsoftGlueZynq.so -shared -fPIC -Wl,-hlibsoftGlueZynq.so -L/opt/epics/synApps/support/softGlueZynq-R2-0-1/lib/linux-x86_64 -L/opt/epics/base/lib/linux-x86_64 -L/opt/epics/synApps/support/asyn-R4-33/lib/linux-x86_64 -L/opt/epics/synApps/support/seq-2-2-5/lib/linux-x86_64           -rdynamic -m64             drvZynq.o devAsynSoftGlue.o devA32Zed.o sampleCustomInterruptHandler.o pixelTriggerAsub.o scalerAsub.o pixelTriggerDmaAsub.o histScalerDmaAsub.o MCSAsub.o clockConfigAsub.o readSoftGlueCounter_ISR.o devSGscaler16.o   -lasyn -lseq -lpv -ldbRecStd -ldbCore -lca -lCom -Wl,-Bdynamic  -lpthread   -lreadline -lm -lrt -ldl -lgcc
/bin/ld: pixelTriggerAsub.o:(.bss+0x0): multiple definition of `myISRData'; sampleCustomInterruptHandler.o:(.bss+0x0): first defined here
/bin/ld: MCSAsub.o:(.bss+0x0): multiple definition of `myISRData'; sampleCustomInterruptHandler.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
```

**Build Environment**
Architecture: linux x86-64 (Debian 11)
Epics Base Version: EPICS Base 3.15.9 + SynApps 6.0
Dependent Module Versions:

**Comment by keenanlang** [comment]
fixed by #7

**Comment by cdn-lnk** [comment]
Thank you guys. =)
